lib/db/mem.js

```diff
@@ -887,6 +890,7 @@ module.exports = function (log, error) {
           account.wrapWrapKb = data.wrapWrapKb
           account.verifierSetAt = data.verifierSetAt
           account.verifierVersion = data.verifierVersion
+          account.profileChangedAt = data.verifierSetAt
```

Comment:
> do we assume kB stays the same through a resetAccount

That statement is obviously wrong since this happens for a password reset. Does `keysChangedAt` need to be updated here?

Reply:
The problem is, we don't know yet - this gets called for both password change and password reset, and even on password reset the user might have used a recovery key to preserve their existing `kB`. We'll need a patch on the auth-server to send in a "did `kB` actually change?" flag to this endpoint.  I was going to do that as a follow-up and create another version of this stored procedure to use it, in the interests of getting this in sync with the applied migrations in prod ASAP.

But if you prefer, I can timebox a quick look at what it would take to make that change on the auth-server at the same time?

Reply:
For completeness, I'll mention that the use of `COALESCE(a.keysChangedAt, a.verifierSetAt, a.createdAt)` and ` item.keysChangedAt = item.keysChangedAt || item.verifierSetAt || item.createdAt` means that we are defaulting `keysChangedAt` to `verifierSetAt` in the absence of this information.

Reply:
> But if you prefer, I can timebox a quick look at what it would take to make that change on the auth-server at the same time?

That sounds like a lot of work, enough for a merge and follow on.